app/views/shared/_status.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="<%= dialog_column_class %>">
+  <h1>Status of your request</h1>
+  <hr />
+  <dl class='dl-horizontal dl-invert'>
+    <dt>Requested on</dt>
+    <dd><%= request.created_at.to_formatted_s(:long_ordinal) %></dd>
+  </dl>
+  <hr />
+  <%= render partial: 'shared/searchworks_item_information', locals: { request: request } %>
```

Comment:
Hm. I stole that example from Spotlight, so we've been able to make it work before. `#current_request` would also work for me.